MSAL/src/configuration/MSALWebviewParameters.m

```diff
@@ -86,4 +86,27 @@ - (id)copyWithZone:(__unused NSZone *)zone
     return item;
 }
 
+#if !MSID_EXCLUDE_WEBKIT
+
++ (WKWebViewConfiguration *)createWebViewConfigWithPKeyAuthUserAgent
+{
+    WKWebViewConfiguration *webConfig = [WKWebViewConfiguration new];
```

Comment:
Should this one use common core code instead of duplicating it?

Reply:
My thought was that we would need to expose the API to the public users by declaring & defining it within the MSAL space. That's what I believe was needed for ADAL. Could you please let me know if that's not the case for MSAL?

Reply:
Exposing public API doesn't mean that we need to duplicate the code :) Public API can just call into common library.

Reply:
Thank you for clarifying. I understand what you mean now.